config-default.yml

```diff
@@ -291,6 +291,11 @@ guild:
         # Streaming
         video:          764245844798079016
 
+        # Patreon
+        patreon_tier_1: 505040943800516611
+        patreon_tier_2: 743399725914390631
+        patreon_tier_3: 743400204367036520
```

Comment:
Aren't these supposed to be the IDs from the actual Python Discord server?

Reply:
assuming the ones that chris posted in #1765 are correct, then these are correct